express-post-json/index.js

```diff
@@ -0,0 +1,31 @@
+const express = require('express');
+const grades = {};
+const app = express();
+
+var gradesArray = [];
+app.get('/api/grades', (req, res) => {
+  for (const grade in grades) {
+    gradesArray.push(grades[grade]);
+  }
+  res.json(gradesArray);
+});
```

Comment:
You'll find that every time you make a `GET` request, the number of grades outputted gets larger.